sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala

```diff
@@ -85,6 +89,14 @@ object JDBCRDD extends Logging {
     }
   }
 
+  /**
+   * Time duration of remote schema discovery
+   */
+  val remoteSchemaDiscoveryMetric: SQLMetric = SQLMetrics.createNanoTimingMetric(
```

Comment:
The `SQLMetric` instance should be one per SparkPlan, but this instance is in `object JDBCRDD`